src/prompt/cursor.rs

```diff
@@ -34,6 +55,24 @@ impl StringCursor {
         }
     }
 
+    pub fn move_word_left(&mut self) {
+        if self.cursor > 0 {
+            let jumps = word_jump_indices(&self.value);
+            let ix = jumps.binary_search(&self.cursor).unwrap_or_else(|i| i);
+            self.cursor = jumps[std::cmp::max(ix - 1, 0)];
```

Comment:
`ix - 1` will panic if `ix == 0`.
```suggestion
            self.cursor = jumps[ix.saturating_sub(1)];
```

Reply:
oh good catch! i'm now realizing that ix will always be >= 1 which is why i never got a panic here. This is due to self.cursor > 0 in the if condition. I'll figure out a way to clean this up

Reply:
done in https://github.com/fadeevab/cliclack/pull/9/commits/ae9b394a0abdf057888b3e3ab0a84dd488f20318